app/views/confirmation/ConfirmationMergeJourneyView.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components.FormWithCSRF
+@import controllers.confirmation.routes._
+
+@this(
+    layout: views.LayoutProvider,
+    formHelper: FormWithCSRF,
+    submitButton: playComponents.submit_button
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@layout(
+    pageTitle = messages("confirmationMergeJourney.title"),
+    showBackLink = false
+) {
+    <div class="govuk-panel govuk-panel--confirmation">
+        <h1 class="govuk-panel__title">@messages("confirmationMergeJourney.heading")</h1>
+    </div>
+
+    <h2 class="govuk-heading-m">@messages("confirmationMergeJourney.whatHappensNext")</h2>
+    <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+            <span class="govuk-visually-hidden">@messages("site.warning")</span>
+            @messages("confirmationMergeJourney.warningMessage")
+        </strong>
+    </div>
+
+    <p class="govuk-body">@messages("confirmationMergeJourney.para")</p>
+
+    @formHelper(action = ConfirmationMergeJourneyController.onSubmit(), Symbol("autoComplete") -> "off") {
+        <button type="submit" class="govuk-button" data-module="govuk-button">
+            @messages("site.continue")
+        </button>
+    }
+}
```

Comment:
You don't need to use a form when you're not actually submitting anything. Can simply use a link with govuk-button class instead (there should be a helper for this). When you use a form for this you can have unintended effects like the csrf token appearing in the url on the next page, not really an issue but shouldn't happen either way.